src/prettyprint.lua

```diff
@@ -0,0 +1,88 @@
+local COLORS = require("src.utils.consoleColor")
```

Comment:
Please use British spellings where possible for consistency.

(Before arguing on this, please see https://github.com/TAServers/lest/graphs/contributors)